modules/messages/server/controllers/messages.server.controller.js

```diff
@@ -280,6 +282,13 @@ exports.send = function(req, res) {
       });
     },
 
+    // Here we collect some message data into the influxdb
+    function (message, done) {
+      // the module returns a promise
+      messageToInfluxService(message, { Message: Message, influxService: influxService })
```

Comment:
Instead of passing Influx and Message schema here, you could just require it inside the service?